### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by AdityaSinghDGR8**

### CODE REVIEW

**Comment by AdityaSinghDGR8** [comment]
@Vaibhavi24 review

**Comment by AdityaSinghDGR8** [comment]
@dotangad nice

File: matrix.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<html>
+	<head>
+	</head>
+	<body>
+		<div id='log'></div>
+		<div id='matrix'></div>
+		<script>
+			var M=[];
```
Comment (contactvaibhavi) [inline matrix.html:8]: Pls give this var a more descriptive name

File: matrix.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
+<html>
+	<head>
+	</head>
+	<body>
+		<div id='log'></div>
+		<div id='matrix'></div>
+		<script>
+			var M=[];
+			function Create(M,N=3) {
+				for (var i=0;i<N;i++)
```
Comment (contactvaibhavi) [inline matrix.html:10]: Pls give this var N a more descriptive name

**Review by contactvaibhavi — APPROVED** [review]
Great code 😊
Like the modular design!
Keep it up 😊

**Pull request merged**